ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js

```diff
@@ -1,8 +1,14 @@
+var value = $('#recaptcha').data('custom-property');
```

Comment:
@nitheshkumar-hcl : Pl add relevant variable name here

Reply:
Changed as per the review comments